sklearn/model_selection/tests/test_validation.py

```diff
@@ -281,6 +281,17 @@ def test_cross_val_score():
                   error_score='raise')
 
 
+@pytest.mark.filterwarnings('ignore:You should specify a value for')  # 0.22
+def test_cross_validate_many_jobs():
+    # regression test for #12154: cv='warn' with n_jobs=-1 trigger a copy of
+    # the parameters leading to a failure in check_cv due to cv is 'warn'
+    # instead of cv == 'warn'.
+    X, y = load_iris(return_X_y=True)
+    clf = SVC(gamma='auto')
```

Comment:
Can we just use a LogisticRegresion? (Trying to gain ms on the test suite run, also just makes sense to use the simplest estimator there is since it's not relevant)